search/search_repository.go

```diff
@@ -263,6 +264,18 @@ func parseMap(queryMap map[string]interface{}, q *Query) {
 		case bool:
 			s := concreteVal
 			q.Negate = s
+		case map[string]interface{}:
+			if v, ok := concreteVal["$IN"]; ok {
+				q.Name = Q_OR
+				c := &q.Children
+				for _, vl := range v.([]interface{}) {
```

Comment:
This is a suggestion, it isn't a blocker per se. As a quick example of how it could be https://play.golang.org/p/1MtYJ8IthV